newsletter/templates/newsletter/message/message.html

```diff
@@ -1,7 +1,6 @@
-{% load thumbnail %}<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN"
-   "http://www.w3.org/TR/html4/strict.dtd">
+{% load thumbnail i18n %}<!DOCTYPE html>

```

Comment:
+2 on that suggestion. Maybe finish this work, commit it and then make another commit changing *all* template headers to HTML5.